flixel/system/FlxSound.hx

```diff
@@ -737,19 +732,55 @@ class FlxSound extends FlxBasic
 		return Volume;
 	}
 
-	#if (sys && openfl_legacy)
 	inline function get_pitch():Float
 	{
 		return _pitch;
 	}
 
 	function set_pitch(v:Float):Float
 	{
-		if (_channel != null)
-			_channel.pitch = v;
+		@:privateAccess
+		#if openfl_legacy
+			#if sys
+			@:privateAccess
+			if (_channel != null)
+				_channel.__source.pitch = v;
+			#else
+			FlxG.log.error("Pitch is supported only in sys systems while using openfl legacy!! switch the build into sys or upgrade your openfl!");
+			return 1;
+			#end
+		#else
+		#if flash
+		FlxG.log.error("Pitch is not supported on Flash!!!");
```

Comment:
Rather than throwing a runtime error on flash we should hide the feature entirely and throw a compile time error when they try to use it